README.md

```diff
@@ -27,17 +27,21 @@ Spring Boot Starter Handlebars will automatically register handlebars helpers ba
 Add any handlebars helper to dependencies and you can start using it.
 ```gradle
 dependencies {
-    compile 'com.github.jknack:handlebars-helpers:2.0.0'
-    compile 'com.github.jknack:handlebars-jackson2:2.0.0'
-    compile 'com.github.jknack:handlebars-humanize:2.0.0'
-    compile 'com.github.jknack:handlebars-markdown:2.0.0'
+    compile 'com.github.jknack:handlebars-helpers:2.0.0',
+            'com.github.jknack:handlebars-jackson2:2.0.0',
+            'com.github.jknack:handlebars-humanize:2.0.0',
+            'com.github.jknack:handlebars-markdown:2.0.0'
 }
 ```
 NOTE: Jackson2Helper and MarkdownHelper will register with name `json` and `md` respectively.
 Every other helper will register with its default name.
 
-More information about helpers can be found on
-[Handlebars.java](https://github.com/jknack/handlebars.java).
+More information about available helpers can be found on
+[Handlebars.java](https://github.com/jknack/handlebars.java#helpers).
+
+### Custom helpers
+
+To register helper use [@HandlebarsHelper](src/main/java/pl/allegro/tech/boot/autoconfigure/handlebars/HandlebarsHelper.java) annotation.
```

Comment:
We could link here to this resources. They document helper methods:
- https://github.com/jknack/handlebars.java/blob/master/handlebars/src/main/java/com/github/jknack/handlebars/HelperRegistry.java
- https://github.com/jknack/handlebars.java#using-a-helpersource